### PR TITLE
initial commit

### PR DESCRIPTION
**Pull request by johnsickels**

### CODE REVIEW

**Comment by johnsickels** [comment]
I am reviewing the changes now

**Review by johnsickels — COMMENTED** [review]
looks good

File: app.js [hunk 1]
```diff
@@ -0,0 +1,2 @@
+console.log("FATAL ERROR");
```
Comment (johnsickels) [inline app.js:1]: I like this part.

**Review by johnsickels — COMMENTED** [review]
sdgkh

**Review by madisonwmoore — COMMENTED** [review]
Good

**Pull request closed without merging**